data-reporting-service/src/main/resources/db/odse/004-fn_get_value_by_cd_codeset-001.sql

```diff
@@ -0,0 +1,15 @@
+CREATE OR ALTER function [dbo].[\
```

Comment:
I think the name of this sproc got removed during commit.

Reply:
@sveselev the function name is missing